File: src/libs/ThirdParty.CSSuppliers/ExpediaRapid/SerializableClasses/Fee.cs
namespace ThirdParty.CSSuppliers.ExpediaRapid.SerializableClasses
{
    using Newtonsoft.Json;

    public class Fee
    {

        [JsonProperty("type")]
        public string Type { get; set; }

        [JsonProperty("value")]
        public decimal Value { get; set; }

        [JsonProperty("currency")]
        public string Currency { get; set; }

        [JsonProperty("scope")]
        public string Scope { get; set; }

        [JsonProperty("frequency")]
        public string Frequency { get; set; }

    }

}

File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Fee.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses

    Public Class Fee

        <JsonProperty("type")>
        Public Property Type As String

        <JsonProperty("value")>
        Public Property Value As Decimal

        <JsonProperty("currency")>
        Public Property Currency As String

        <JsonProperty("scope")>
        Public Property Scope As String

        <JsonProperty("frequency")>
        Public Property Frequency As String

    End Class

End Namespace
